templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+{% extends "layouts/main.html" %}
+
+{% block content %}
+
+    {% with page = "home"  %}
+        {% set banner_background_class= "banner-image-home" %}
+        {% set banner_large_background_class = "banner-image-home-large" %}
+        {% set banner_title_class = "banner-title-home" %}
+        {% set banner_video = "video/banner-video.mp4" %}
+        {% set page_title = "Data Analytics for Canadian Climate Services, STAC API" %}
+        {% set page_subtitle = "Climate Data Search" %}
+        {% set page_slogan = "Explore this rich repository of information about Earth’s climate—uncovering insights into its past, understanding current trends, and anticipating future changes." %}
+        {% include "partials/banner.html" %}
+    {% endwith %}
+
+<!--Page content area-->
+<div class="row">
+    <div class="container-fluid content-search-main">
+        <div class="content-search-left">
+        <div id="backButtonContainer" class="div-back-button-container">
+            <i class="fa-solid fa-arrow-left back-button-arrow"></i><a id="backButton" class="button-back-button subtitle-1" role="button">Home</a><img  src="images/open-in-app.svg">
+        </div>
+
+            {% with %}
+                {% set dropdown_type = "regular_dropdown_source" %}
+                {% set dropdown_button_id = "dropdownSourceRegular" %}
+                {% set dropdown_label_text = "Source" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+            {% with %}
+                {% set dropdown_type = "regular_dropdown_share" %}
+                {% set dropdown_button_id = "dropdownShareRegular" %}
+                {% set dropdown_label_text = "Share" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+        <div id="advancedFilters" class="div-advanced-filters">
+            <div>
+                <h6 class="title-advanced-filters">Advanced Filters</h6>
+            </div>
+
+            {% with %}
```

Comment:
Why does the time frame dropdown have a box shadow but none of the others do?

![image](https://github.com/user-attachments/assets/c5911be6-dc52-4e41-be28-b47c72692be9)